@Types/modules.d.ts

```diff
@@ -16,14 +16,13 @@ declare module 'fast-isnumeric' {
 }
 
 declare class Remarkable {
-    constructor();
     render(value: string): any;
 }
 
 declare interface RemarkableCtor {
-    new(): Remarkable;
+    new(options?: any): Remarkable;
```

Comment:
I think that with `@types/Remarkable` added, all of the remarkable definitions becomes redundant.